apps/allocations/contracts/Allocations.sol

```diff
@@ -207,10 +212,14 @@ contract Allocations is AragonApp, Fundable {
     *      to be called by a RangeVote (options get weird if it's not)
     *      but for our use case the “SET_DISTRIBUTION_ROLE” will be given to
     *      the RangeVote.
-    * @notice Sets the distribution for the given `payoutId` using an the
-    *         supplied candidate keys and support values.
-    * param _candidateKeys The array of keys for all candidates in this payout
-    * param _supports The Array of all support values for the various candidates
+    * @notice Sets the distribution proportion for this allocation
+    * @param _candidateAddresses Array of candidates to be allocated a portion of the payouut
+    * @param _supports The Array of all support values for the various candidates. These values are set in range voting
+    * @param _payoutId The Account used for the payout
+    * @param _informational boolean used to indicate whether and funds will be trnsacted onchain
+    * @param _recurring boolean used to indicate whether this is a recurring or one-time payout
+    * @param _period time interval between each recurring payout
+    * @param _amount The quantity of funds to be allocated
     */
```

Comment:
Where in the UI is this?

Reply:
This notic appears when you click "New Allocation" on the Allocation account card

Reply:
Line 215 in the new changes is what will be displayed: https://github.com/AutarkLabs/planning-suite/pull/470/files/773673273cf237aed9ea8387398b1a1e877c6e00#diff-d4d9bf65696fe98cabb21de714647055R215